codegen-renderers/src/main/kotlin/io/vrap/codegen/languages/java/model/second/JavaModelModuleSecond.kt

```diff
@@ -0,0 +1,18 @@
+package io.vrap.codegen.languages.java.model.second
+
+import com.google.inject.AbstractModule
+import com.google.inject.multibindings.Multibinder
+import io.vrap.codegen.languages.java.model.JavaStringTypeRenderer
+import io.vrap.rmf.codegen.rendring.ObjectTypeRenderer
+import io.vrap.rmf.codegen.rendring.StringTypeRenderer
+
+class JavaModelModuleSecond: AbstractModule() {
```

Comment:
How about calling this `JavaModelWithInterfacesModule`? This reflects better what this module actually does ;-)

Reply:
Agree. Totally forgot to change the name.